Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM python:2-onbuild
+MAINTAINER James Swanick "swanijam@gmail.com"
+
+WORKDIR /app/
+ADD site.py site.py
+ADD templates templates/
+
+RUN echo $(ls)
+RUN mkdir tmp/
+RUN virtualenv venv
```

Comment:
No need for a virtual environment since there is nothing else is running in this container.